.github/workflows/web-test-runner.yml

```diff
@@ -0,0 +1,67 @@
+name: Run Web Test Runner integration tests
+
+on:
+    push:
+        branches:
+            - master
+            - release
+            - 'spring*'
+            - 'summer*'
+            - 'winter*'
```

Comment:
Might be better to just run on `master` at first, since older branches won't have the tweaks that are necessary for the Karma-to-WTR migration. We can still run Karma locally for back-ported bug fixes.